### ISSUE TITLE
Figured my own problem(s) out. Proxy plugins make SelfControl ineffective and you must also clear all cookies in addition to cache :/

### ISSUE THREAD

**Issue by rellimmot** [SelfControlApp/selfcontrol#118]
**_TL;DR:**_

---

I found the issue with Firefox. I had accidentally left my 'Stealthy' proxy plugin enabled. SelfControl is useless if I turn on the Stealthy plugin.
That said on the topic of proxies, I also tried TorBrowser and found SelfControl to be ineffective. 
Not that the average end-user is going to be using proxy plugins and browsers, but I wish I PERSONALLY could use SelfControl because I in fact have none. I guess I'll have to uninstall the plugins during my social media blackout blitz (1 week).

_Also, note:_ If you use GasMask to edit your hosts file, SelfControl seems to be ineffective. Close GasMask while using SelfControl.

---

> This is originally a comment in closed issue #73, but I'll open it as a new one since I've also found SelfControl (and even /hosts) to be useless on FireFox 20 (see Update's at bottom)
> 
> Still not working for me. I just checked for updates, I'm up to date. Err... Blocks anything but Facebook
> I've even tried adding
> 127.0.0.1 www.facebook.com
> 127.0.0.1 facebook.com
> 127.0.0.1 static.ak.fbcdn.net
> 127.0.0.1 www.static.ak.fbcdn.net
> 127.0.0.1 login.facebook.com
> 127.0.0.1 www.login.facebook.com
> 127.0.0.1 fbcdn.net
> 127.0.0.1 www.fbcdn.net
> 127.0.0.1 fbcdn.com
> 127.0.0.1 www.fbcdn.com
> 127.0.0.1 static.ak.connect.facebook.com
> 127.0.0.1 www.static.ak.connect.facebook.com
> 127.0.0.1 login.facebook.com
> 127.0.0.1 www.facebook.com
> 127.0.0.1 blog.facebook.com
> 127.0.0.1 apps.facebook.com
> to my hosts file, still doesn't block. Facebook is a sneaky prick! Something to do with it being https? Using Chrome in OS X 10.8.2
> 
> **UPDATE:
> I opened an "Incognito Browsing" window and tried facebook.com; didn't work. Seems you must clear cookies to effectively block Facebook! Cleared cookies, Facebook is dead. Freedom! Horpfully that give's y'all some lead to finding a workaround to blocking FB without people having to clear their cookies- which is slightly annoying. I know I could have gone in and singled out the FB cookie, but not all users are that savvy. May want to mention this in the FAQ, at least.
> 
> ***UPDATE:
> Out of curiosity I closed Chrome, reopened, and I was able to navigate to Facebook.
> Went on a war path, cleared ALL BROWSER DATA from all 3 browsers (Firefox, Safari, Chrome).
> Started a new timer, blacklisting facebook.com, plus twitter.com and reddit.com as controls
> 
> In Firefox: SelfControl seems to be useless. I can access facebook.com and my controls, twitter.com and reddit.com without issues. Thing about FireFox is I cant even block websites from it with my /hosts file. Its on some next level Bill Clinton shit.
> In Chrome: can access facebook.com SOMETIMES?, twitter.com is blocked, reddit.com is blocked
> In Safari: can not access any. great success.
> 
> Am I doing something wrong here? I'm no diehard programmer, but I'm certainly lightyears ahead of Jim Blowstick who just bought his first computer.
> 
> ****UPDATE (AGAIN!):
> 
> I found the issue with Firefox. I had accidentally left my 'Stealthy' proxy plugin enabled. SelfControl is useless if I turn on the Stealthy plugin. Works in FireFox just fine (blocks all blacklisted sites) with Stealthy disabled. 
> 
> That said on the topic of proxies, I tried TorBrowser and found SelfControl to be ineffective. 
> 
> Not that the average end-user is going to be using proxy plugins and browsers, but I wish I PERSONALLY could use SelfControl because I in fact have none. I guess I'll have to uninstall the plugins during my social media blackout blitz (1 week).
> 
> Also, note: If you use GasMask to edit your hosts file, SelfControl seems to be ineffective. Close GasMask while using SelfControl (yes, GasMask was closed during all the testing ranted above).
> .
